### ISSUE TITLE
Reproducing paper results

### ISSUE THREAD

**Issue by fe-silva** [perslev/U-Time#10]
Hi,

I'm trying to reproduce your paper results, but I'm achieving slight lower scores. The mean dice that I achieve on S-EDF-153 was 0.7091. In the paper, the mean dice reported is 0.76

I would like to confirm the paper protocol: 
- The splits fractions used were train (0.6), test(0.2), val(0.2), correct?
- Were the results obtained using one_shot evaluation?

Do you have any insights what could be causing this decrease of performance?
Thanks

**Comment by perslev** [comment]
Hi fe-silva,

Thanks for trying out U-Time! The evaluation score that you have observed is definitely significantly lower than those reported in the paper. I would be happy to help you gain some insight as to why that may be.

The main difference is likely a result of your experiment being a fixed 60/20/20 split, whereas the number you refer to from the paper comes from a 10-fold CV experiment. I now realize that it is confusing that the "TLDR" example here on github does not actually produce a result from the paper. It's intention was simply to show how one may run a quick experiment, wheras doing the full 10-CV can be a heavy comoutational task for many users.

In the 10-CV setting the training set size in each split is significantly larger than what you have used, which may be the reason your model performs worse.

That being said, the score is still a bit lower than I would expect, and lower than what I remember from last time I did this experiment myself. If you would be so kind to share some additional details on e.g. what versions of tensorflow, Cuda etc you have used, then I will try my best to find time to run an equivalent experiment myself to check this out.

Yes, we used one-shot evaluation in the paper.

Finally, is the score that you show here the output of the command "ut cm" or "ut summary"? If from the later, note that such scores are mean values computed across per-subject metrics (as reported in supplementary table S.4 of the paper), while scores from Table 2 are global dice metrics computed across all subjects in the dataset. Those scores are likely higher on average.

P.S. here I assumed that you used our implementation to run this experiment. Please let me know if you re-implemented the model or training loop yourself.

Kind regards
Mathias

**Comment by fe-silva** [comment]
Hi Mathias,

Thanks for your answer and tips!
I think that your points explains my results.

Yes, I'm using your code with tensorflow 2.1. 
The score I mentioned was the "Grand mean" dice score, output from "ut evaluate", which is similar to the output of "ut summary". And I was comparing it with the score from Table 2. 
In Table S.4, the mean dice is near 0.718, which is much closer to what I observed, and using "ut cm", I got 0.74.

The other point that you mentioned about having more subjects with CV will probably explain the remaining difference. By the way, this part is clear on the paper. (my mistake).

I'll run again using 10CV and I let you know about it.

Once again, thank you!

Kind Regards,
Fernanda
